### PR TITLE
scripts: update provisioning for el8 OS family

### LINKED ISSUE #1356: install-build-deps script fails on CentOS 8
The `install-build-deps` script mentioned in [QSG](https://github.com/Seagate/cortx-motr/blob/409f711fcfa22b393e34a14f49eab5fa142131bb/doc/Quick-Start-Guide.rst#build) fails on CentOS 8:

```
[seagate@centos8 motr]$ sudo scripts/install-build-deps
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:42:37 ago on Fri 31 Dec 2021 00:11:27 GMT.
No match for argument: python-ipaddress
Error: Unable to find a match: python-ipaddress
```

### PR DESCRIPTION
**Pull request by andriytk**
Closes #1356.

# Problem Statement
`scripts/install-build-deps` does not work on CentOS 8 or Rocky Linux

# Design
Update the scripts.

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/provisioning/roles/lustre-client/tasks/main.yml [hunk 1]
```diff
@@ -55,6 +56,7 @@
     - { version: 2.12.4, enabled: '{{ (redhat_release is version("7.7", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.5, enabled: '{{ (redhat_release is version("7.8", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.6, enabled: '{{ (redhat_release is version("7.9", ">=")) | ternary("yes", "no") }}' }
```
Comment (madhavemuri) [inline scripts/provisioning/roles/lustre-client/tasks/main.yml:58]: s/"7.9", ">="/"7.9", "="

**Comment by andriytk** [comment]
I see this error now:

```
TASK [motr-build : install local libfabric rpms] ***************************************************************************
task path: /home/ant/cortx-motr/scripts/provisioning/roles/motr-build/tasks/main.yml:350
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c 'echo ~root && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp `"&& mkdir "` echo /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766 `" && echo ansible-tmp-1641485716.4352906-341996-68034374903766="` echo /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766 `" ) && sleep 0'
Using module file /usr/lib/python3.6/site-packages/ansible/modules/packaging/os/dnf.py
<localhost> PUT /root/.ansible/tmp/ansible-local-298918ocrdtd5p/tmpszlto95h TO /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py
<localhost> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/ /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/libexec/platform-python /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py", line 102, in <module>
    _ansiballz_main()
  File "/root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py", line 94, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py", line 40, in invoke_module
    runpy.run_module(mod_name='ansible.modules.packaging.os.dnf', init_globals=None, run_name='__main__', alter_sys=True)
  File "/usr/lib64/python3.6/runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py", line 1370, in <module>
  File "/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py", line 1359, in main
  File "/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py", line 1338, in run
  File "/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py", line 986, in ensure
  File "/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py", line 885, in _install_remote_rpms
  File "/usr/lib/python3.6/site-packages/dnf/base.py", line 1179, in add_remote_rpms
    raise IOError(_("Could not open: {}").format(' '.join(pkgs_error)))
OSError: Could not open: /root/rpmbuild/RPMS/x86_64/libfabric-1.11.2-1.el7.x86_64.rpm /root/rpmbuild/RPMS/x86_64/libfabric-devel-1.11.2-1.el7.x86_64.rpm
fatal: [localhost]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-tmp-1641485716.4352906-341996-68034374903766/AnsiballZ_dnf.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible.modules.packaging.os.dnf', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code\n    mod_name, mod_spec, pkg_name, script_name)\n  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py\", line 1370, in <module>\n  File \"/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py\", line 1359, in main\n  File \"/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py\", line 1338, in run\n  File \"/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py\", line 986, in ensure\n  File \"/tmp/ansible_dnf_payload_mtzv6l8j/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py\", line 885, in _install_remote_rpms\n  File \"/usr/lib/python3.6/site-packages/dnf/base.py\", line 1179, in add_remote_rpms\n    raise IOError(_(\"Could not open: {}\").format(' '.join(pkgs_error)))\nOSError: Could not open: /root/rpmbuild/RPMS/x86_64/libfabric-1.11.2-1.el7.x86_64.rpm /root/rpmbuild/RPMS/x86_64/libfabric-devel-1.11.2-1.el7.x86_64.rpm\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
msg:  MODULE FAILURE
See stdout/stderr for the exact error

PLAY RECAP *****************************************************************************************************************
localhost                  : ok=69   changed=5    unreachable=0    failed=1    skipped=13   rescued=0    ignored=1   

```

File: scripts/provisioning/roles/lustre-client/tasks/main.yml (hunk 1 above)
Comment (andriytk) [reply scripts/provisioning/roles/lustre-client/tasks/main.yml]: Fixed. Thanks!

**Comment by andriytk** [comment]
Tested on CentOS 8.3 and Rocky Linux 8.5 - works fine:
```
PLAY RECAP *****************************************************************************************************************
localhost                  : ok=74   changed=5    unreachable=0    failed=0    skipped=15   rescued=0    ignored=1   
```

**Comment by madhavemuri** [comment]
PLAY RECAP *************************************************************************************************************************************************
localhost                  : ok=74   changed=5    unreachable=0    failed=0    skipped=15   rescued=0    ignored=1

 cortx-motr]# cat /etc/redhat-release
CentOS Stream release 8
 cortx-motr]#

PLAY RECAP ***************************************************************************************************************************************************************
localhost                  : ok=74   changed=5    unreachable=0    failed=0    skipped=15   rescued=0    ignored=1

cortx-motr]# cat /etc/redhat-release
Rocky Linux release 8.4 (Green Obsidian)

`./scripts/install-build-deps` completed successfully on CentOS Steam 8 and Rocky Linux 8.4

**Review by madhavemuri — APPROVED** [review]

**Pull request merged**